concordium-node/README.md

```diff
@@ -1,9 +1,9 @@
 # Concordium node implementation
 ## Dependencies to build the project
-* Rust (stable 1.45.2 for using static libraries)
+* Rust (stable 1.53 for using static libraries)
 * binutils >= 2.22
 * cmake >= 3.8.0
-* [flatc](http://google.github.io/flatbuffers/flatbuffers_guide_building.html) v1.11 and v1.12 are known to work. (build using CMake and copy to `~/.local/bin`)
+* [flatc](http://google.github.io/flatbuffers/flatbuffers_guide_building.html) v2.0 is known to work. (build using CMake and copy to `~/.local/bin`)
```

Comment:
It might be useful to link to the release page for 2.0.0, where they have binaries available: https://github.com/google/flatbuffers/releases/tag/v2.0.0
(I don't know about this situation on other platforms, but msys2 and chocolatey don't currently have this version available.)

Reply:
Added.